util/dashboard/gen_dashboard_entry.py

```diff
@@ -63,17 +62,17 @@ def get_doc_url(base, url):
     """
     assert isinstance(url, str) and len(url) > 0
     if url[0] == '/':
-        return url
+        return '/book' + url
     else:
-        return '/' + base + '/' + url
+        return '/book/' + base + '/' + url
 
 
 # Link module name with its design spec doc.
 def get_linked_design_spec(obj):
     result = ""
     if 'design_spec' in obj:
         result = "<span title='Design Spec'><a href='{}'>".format(
-            get_doc_url(obj['_ip_desc_hjson_dir'], obj['design_spec']))
+            get_doc_url(obj['_ip_desc_hjson_dir'], obj['design_spec']) + "/..")
```

Comment:
Is it possible to drop the last component of `obj['design_spec']` wherever it is created, rather than appending `/..` here?

(Same for subsequent `get_linked_*()` functions.)

Reply:
These 'obj' fields come from the .hjson for each block, and these fields ('design_spec', 'dv_doc', 'hw_checklist') don't seem to be used for much else so I think it would be nice to re-evaluate changing their meaning to be more-target/cleaner for the use here. Maybe as a follow-up task?

Reply:
Yeah, I'd rather this was a follow up task